.github/workflows/update-copyright-years-in-license-file.yml

```
@@ -0,0 +1,17 @@
+name: Update copyright year in license file
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 1 1 *"
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)